command/loadbot/helper.go

```diff
@@ -52,18 +52,51 @@ func estimateGas(client *jsonrpc.Client, txn *types.Transaction) (uint64, error)
 	return gasEstimate, nil
 }
 
-// calcMaxTimeout calculates the max timeout for transactions receipts
-// based on the transaction count and tps params
-func calcMaxTimeout(count, tps uint64) time.Duration {
-	waitTime := minReceiptWait
-	// The receipt timeout should be at max maxReceiptWait
-	// or minReceiptWait + tps / count * 100
-	// This way the wait time scales linearly for more stressful situations
-	waitFactor := time.Duration(float64(tps)/float64(count)*100) * time.Second
-
-	if waitTime+waitFactor > maxReceiptWait {
-		return maxReceiptWait
```

Comment:
Why did you remove this piece of logic?

We purposefully placed it here so the receipt wait times for transactions would be calculated dynamically on the fly, instead of relying on a single default value / passed in by the user.

Why not just expand this method to account for receipt wait times on contract deployments / calls?

Reply:
Restored `calcMaxTimeout` 6e26304396b9c7250e1938be83405f412e4d7e96 and increased the minimum time to wait for receipts to 1min. 3c8fa5a904c5428e790ac7668be99c7f213e07ef .
Also, with the `--max-wait` flag the user can define how many minutes to wait for receipts, as for some modes like `erc721` and a large number of txns, the `calcMaxTimeout` timeout is too short.